src/index.test.ts

```diff
@@ -1666,3 +1666,52 @@ test('Should set sdkState to healthy when client is started', (done) => {
         done();
     });
 });
+
+describe('READY event emission', () => {
+    let client: UnleashClient;
+
+    const config: IConfig = {
+        url: 'http://localhost/test',
+        clientKey: '12',
+        appName: 'web',
+        bootstrap: [
+            {
+                enabled: false,
+                name: 'test-frontend',
+                variant: { name: 'some-variant', enabled: false },
+                impressionData: false,
+            },
+        ],
+        bootstrapOverride: false,
```

Comment:
Is this a new configuration? Not sure if boostrap override is a name that brings a lot of clarity... but maybe it's an existing setting

Reply:
No, it is an option that was there